app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -90,203 +88,86 @@ For safeguarding reasons, schools will assess your suitability to teach from a r
-
-For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/). 
-
-Most teaching vacancies are advertised in March, April or the first half of May, with jobs starting at the beginning of the school year in September. Set up a [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) job alert to be updated about opportunities throughout the year.
-
-### Other teaching job options
-
-[Supply teachers](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher) fill in when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school.  
-
-[Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
-
-### Tips on applying 
+### Your professional qualifications
 
-As part of your application, schools will usually ask for your:
-
-* educational and professional qualifications
-* employment history
-* personal statement
-* references
-
-Your personal statement needs to cover your teaching approach, your experience and how you would contribute to the life of the school. 
-
-[Get tips on writing a personal statement](https://www.tes.com/jobs/careers-advice/application-and-interview/how-write-perfect-teaching-personal-statement).
-
-Some schools may ask for a curriculum vitae (CV) – [get tips on writing a CV](https://nationalcareers.service.gov.uk/careers-advice/cv-sections).
-
-If you're shortlisted, you’ll be invited for interview and asked to teach part of a class. If the school makes you a conditional offer, you’ll then have to pass [safeguarding and criminal records checks](#what-youll-need-to-pass-safeguarding-and-criminal-records-checks).
-
-### Salaries and benefits
-
-Your salary as a teacher depends on whether you have QTS, and your level of experience.
-
-[Find out more about teacher pay and benefits](/is-teaching-right-for-me/teacher-pay-and-benefits) in England.
-
-[Find out more about teaching in England if you trained overseas](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
+UK ENIC can give you a [statement of comparability](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your academic and professional qualifications compare to their English equivalent. However, you'll need scans or photographs of your original documents. If you're applying to train to teach, you can [get your statement of comparability for free](/non-uk-teachers/train-to-teach-in-england-as-an-international-student#check-your-eligibility).
```

Comment:
@laurahermionetennant92 where it says they can get it for free is this because they're from Ukraine or because once you apply it's free?

Reply:
Everyone can get it for free through a TTA if the provider asks for one when they apply